internal/verifierapi/fees.go

```diff
@@ -81,3 +84,66 @@ func (v VerifierApi) GetPluginPolicyFees(policyId uuid.UUID) (FeeHistoryDto, err
 
 	return feeHistory.Data, nil
 }
+
+// TODO add auth
+func (v VerifierApi) GetPublicKeysFees(ecdsaPublicKey string) (FeeHistoryDto, error) {
+	url := fmt.Sprintf("/fees/publickey/%s", ecdsaPublicKey)
+	v.logger.Debug("Getting public key fees for public key: ", ecdsaPublicKey)
+	v.logger.Debug("URL: ", url)
+	response, err := v.get(url)
+	if err != nil {
+		v.logger.WithError(err).Error("failed to get public key fees")
+		return FeeHistoryDto{}, fmt.Errorf("failed to get public key fees: %w", err)
+	}
+
+	//TODO - this probably shouldn't be a 404, just an empty wrapped response with 0
+	if response.StatusCode == http.StatusNotFound {
+		v.logger.WithError(fmt.Errorf("public key not found")).Error("public key not found for id: ", ecdsaPublicKey)
+		return FeeHistoryDto{}, fmt.Errorf("public key not found")
+	}
+
+	if response.StatusCode != http.StatusOK {
+		v.logger.WithError(err).Error("failed to get public key fees")
+		fmt.Println(response.StatusCode)
```

Comment:
We have v.logger , let's keep using it , don't use `fmt.Println`  , please clean up a few other places.